abis/linux/in.h

```diff
@@ -39,11 +39,44 @@ struct sockaddr_in6 {
 	uint32_t        sin6_scope_id;
 };
 
+struct ip_mreq {
+	struct in_addr imr_multiaddr;
+	struct in_addr imr_interface;
+};
+
+struct ip_mreq_source {
+	struct in_addr imr_multiaddr;
+	struct in_addr imr_interface;
+	struct in_addr imr_sourceaddr;
+};
+
 struct ipv6_mreq {
 	struct in6_addr ipv6mr_multiaddr;
 	unsigned        ipv6mr_interface;
 };
 
+struct in_pktinfo {
+	unsigned int ipi_ifindex;
+	struct in_addr ipi_spec_dst;
+	struct in_addr ipi_addr;
+};
+
+struct in6_pktinfo {
+	struct in6_addr ipi6_addr;
+	uint32_t ipi6_ifindex;
```

Comment:
My local Linux headers define this as an `int`. Is this correct?

Reply:
Is this the correct header to define this? On my local system, this seems to be defined in `<linux/ipv6.h>`